kinetic/distribution.yaml

```diff
@@ -1145,6 +1145,15 @@ repositories:
     source:
       type: git
       url: https://github.com/wg-perception/object_recognition_ros_visualization.git
```

Comment:
This insertion resulted in a `source` entry with no `version` in `object_recognition_ros_visualization`. Please fix.